core/src/main/java/com/datastax/oss/driver/internal/core/config/typesafe/TypesafeDriverConfigProfile.java

```diff
@@ -181,7 +181,7 @@ void register(Derived derivedProfile) {
   }
 
   /**
-   * A profile that was copied from another profile programatically using {@code withXxx} methods.
+   * A profile that was copied from another profile programmatically using {@code withXxx} methods.
```

Comment:
👍 , but it feels a bit weird in this PR since this class is not impacted by any other change (don't remove it, but let's fix unrelated typos in separate commits in the future).